src/order/rel_iso.lean

```diff
@@ -165,6 +165,15 @@ coe_fn_inj (funext h)
 theorem ext_iff {f g : r ↪r s} : f = g ↔ ∀ x, f x = g x :=
 ⟨λ h x, h ▸ rfl, λ h, ext h⟩
 
+/-- The composition of two relation embeddings is a relation embedding. -/
+@[trans] protected def comp (g : s ↪r t) (f : r ↪r s) : r ↪r t :=
```

Comment:
Is this the same as [`rel_embedding.trans`](https://leanprover-community.github.io/mathlib_docs/order/rel_iso.html#rel_embedding.trans)? If so, maybe `rel_hom.comp` should be renamed to `rel_hom.trans` (not necessarily in this PR).

Reply:
Nice find, I had missed `rel_embedding.trans`. It is _almost_ the same except it looks to me like: `f.trans g = g.comp f`. So I think we should not be renaming `rel_hom.comp` but we should perhaps redefine `rel_embedding.trans` in terms of `rel_embedding.comp`?

Reply:
Alternatively, it's a bit gross having `trans` and `comp`, so maybe I could just drop `comp` and use `trans`?

Reply:
Good question, I created [a Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/comp.20vs.20trans).

Reply:
I have just pushed a commit dropping `rel_embedding.comp` since:

- I no longer need it after realising `rel_hom.well_founded` exists,
- we already have `rel_embedding.trans`.